tests/integration/test_methodologies_para_ai_rules_heuristics.py

```
@@ -0,0 +1,1639 @@
+        )
+
+        with pytest.raises(ValueError, match="subconditions"):
+            RuleCondition(type=ConditionType.COMPOSITE)
+
+    def test_non_composite_without_values_raises(self) -> None:
+        from file_organizer.methodologies.para.rules.engine import (
+            ConditionType,
+            RuleCondition,
+        )
+
+        with pytest.raises(ValueError):
```

Ruff (0.15.9)

[warning] `pytest.raises(ValueError)` is too broad, set the `match` parameter or use a more specific exception

(PT011)